src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -0,0 +1,23 @@
+package seedu.duke.parser;
+
+import java.util.Scanner;
+
+/**
+ * To make sense of user commands by extracting keywords, descriptions and time/date
+ */
+public class Parser {
+    public static String getUserInput(Scanner in) {
+        String input;
+        input = in.nextLine();
+
+        return input;
+    }
```

Comment:
Perhaps this could be part of the Ui instead? it does make sense for it to be in Parser class too so there's no need to change it, just that the example code structure had it in the Ui class

Reply:
Thanks for the suggestion. I thought that it would be better to standardize all the print messages in `Ui` class.  Perhaps, should I just convert this to inline under `run( )` in `Duke.java`

Reply:
Yup it does make sense, I don't think anything shd be changed in that case.